app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command = command.split()
```

Comment:
Consider using unpacking to assign values from the `command` list to individual variables. This is more efficient and aligns with the checklist recommendation: `source_path, destination_path = command[1], command[2]`.